### PR TITLE
Add vm:consume to an example and fix nav file.

### PR DESCRIPTION
**Pull request by dhirsch1**
I have added a vm:consume operation to the example, but I know the example does not match the description because it has a Listener operation in it. Please let me know how to update the description or the example so they are both correct.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -259,7 +259,7 @@
 * xref:validation/validation-connector.adoc[Validation Module]
  ** xref:validation/validation-documentation.adoc[Validation Connector Reference]
 * xref:vm/vm-connector.adoc[VM Connector]
- ** xref:vm/vm-publish-listen.adoc[Example: Publish and Get a Response in the VM Connector]]
+ ** xref:vm/vm-publish-listen.adoc[Example: Publish and Listen to Messages with the VM Connector]]
```
Comment (ndinu) [inline modules/ROOT/nav.adoc:262]: Why is this necessary? Which concerns did you have regarding the old tittle?

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 2]
```diff
@@ -33,24 +30,34 @@ So, you can create a VM configuration in a domain like in this:
...
 ----
 <flow name="crossAppRequester">
 	<vm:publish queueName="sharedQueue" config-ref="sharedVMConfig" />
 </flow>
 ----
-
-Notice how both the listener and publisher point to the same configuration defined in the domain.
\ No newline at end of file
++
+. Pull messages from the queue after they are published.
```
Comment (ndinu) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc:91]: I wouldn't say that this is another step, you either put a listener in the queue, or you consume them via the consume operation. this makes it seem like it would be valid to do both.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (dhirsch1) [reply modules/ROOT/nav.adoc:262]: The old title there twice. So I changed the first occurrence to reflect the example the link to that example in the See Also section.(There is an error in the new ink that I need to correct.) Here is the duplicate occurrence: 
Screen Shot 2019-08-26 at 9 10 29 AM



Screen Shot 2019-08-26 at 9 11 40 AM

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/vm/vm-publish-across-apps.adoc:91]: Please tell me how to get the example to be correct. Would the example be correct if I eliminate the listener code frlom it?

Comment (ndinu) [reply modules/ROOT/pages/vm/vm-publish-across-apps.adoc:91]: You could say that instead of using a listener to process the messages you can use the consume operation to consume them.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (ndinu) [reply modules/ROOT/nav.adoc:262]: Great 👍

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 3]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: VM, queues, connector, publish, listen, response, domain, across applications
 
+The following examples show two ways to send messages across different apps:
 
+* Use a listener to receive the messages from the first app and a `publish` operation to feed the received messages to a second app.
+* Use a `publish` operation in a second app to pull messages from the queue of messages, and a `consume` operation to consume the messages.
```
Comment (ndinu) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Use a `publish` operation in a second app to push messages to the queue of messages, and a `consume` operation to consume the messages.

**Review by ndinu — CHANGES_REQUESTED** [review]

**Review by ndinu — APPROVED** [review]

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 4]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
 :keywords: VM, queues, connector, publish, listen, response, domain, across applications
 
+The following examples show two ways to send messages across different apps:
 
+* Use a listener to receive the messages from the first app and a `publish` operation to feed the received messages to a second app.
+* Use a `publish` operation in a second app to push messages to the queue of messages, and a `consume` operation to consume the messages.
 
+In both cases, the two apps must be in the same domain.
 
+== Publishing Messages to a Second App
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Salesforce style says to not use gerunds in headings

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 5]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
 
-Instead of using the `<vm:listener>` message source, you can consume messages in a VM queue through the `<vm:consume>` operation.
+To publish messages to a second app:
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: This statement isn't needed -- just duplicates the heading. please remove.

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 6]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
 
-This operation can consume messages on demand, allowing you to:
+. Create a VM configuration in a domain, like this:
++
+[source,xml,linenums]
+----
+<mule-domain xmlns="http://www.mulesoft.org/schema/mule/domain"
+  xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Please add a carriage return between "/vm http" so this line doesn't require users to scroll the code example.

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 7]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
+  http://www.mulesoft.org/schema/mule/domain http://www.mulesoft.org/schema/mule/domain/current/mule-domain.xsd">
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Same issue here -- forcing users to have a scroller for non-essential info causes angst

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 8]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
+. Publish messages from a second app in the same domain:
++
+[source,xml,linenums]
+----
+<flow name="crossAppRequester">
+  <vm:publish queueName="sharedQueue" config-ref="sharedVMConfig" />
+</flow>
+----
+
+In this example, the listener and `publish` operation point to the same queue (`sharedQueue`) and the same configuration (`sharedVMConfig`).
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Either add tick marks to listener or remove them from publish. And "operation" should be plural.

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 9]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
 [source,xml,linenums]
 ----
 <mule-domain xmlns="http://www.mulesoft.org/schema/mule/domain"
-             xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
-             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-             xsi:schemaLocation="http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
-               http://www.mulesoft.org/schema/mule/domain http://www.mulesoft.org/schema/mule/domain/current/mule-domain.xsd">
+  xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Add carriage returns so example doesn't require scrolling right.

File: modules/ROOT/pages/vm/vm-publish-across-apps.adoc [hunk 10]
```diff
@@ -4,53 +4,96 @@ include::_attributes.adoc[]
...
-	<vm:publish queueName="sharedQueue" config-ref="sharedVMConfig" />
+<flow name="queueConsume">
+  <vm:consume queueName="sharedQueue" config-ref="sharedVMConfig" />
+  <logger />
 </flow>
 ----
 
-Notice how both the listener and publisher point to the same configuration defined in the domain.
\ No newline at end of file
+In this example, the `publish` and `consume` operations point to the same queue (`sharedQueue`) and the same configuration (`sharedVMConfig`).
```
Comment (george-hoffer) [inline modules/ROOT/pages/vm/vm-publish-across-apps.adoc]: Either init cap publish and consume or remove the tick marks. Make sure you check these operation names in Studio to be sure they exist and for capitalization.

**Review by george-hoffer — CHANGES_REQUESTED** [review]

**Review by george-hoffer — APPROVED** [review]
